docs/tutorial/getting_started.md

```diff
@@ -0,0 +1,77 @@
+# Tutorial
+
+```eval_rst
+.. contents::
+```
+
+This guide is meant to give you a simple overview of how to use opsdroid, and to kickstart you projects. It is in no way complete, and I highly recommend checking out the docs for more advanced usage, and a more in-depth overview.
+
+## Getting Started
+
+### What is Opsdroid
+
+Opsdroid is a framework to make creating and extending your ChatOps workflows powerful but simple. Opsdroid is a modular framework. It's event loop is power by the `asyncio` framework. The event loop starts with connectors, which allow Opsdroid to interface with services like Slack and Telegram, then Opsdroid sends the event to parsers and matchers, which will determine which skill to run.
+
+### Installing Opsdroid
+
+```bash
+pip install opsdroid
+```
+
+For Windows and Docker installation, see the installation page.
+
+### Using Opsdroid
+
+#### Configuration
+
+As we mentioned earlier, Opsdroid is modular and built on a event loop. Connectors, parsers, and skills are setup in the `configuration.yaml` file, which is the backbone of any Opsdroid project.
+
+First, we'll setup a shell connector. To do this add a list of connectors, then add shell.
+
+```yaml
+connectors:
+  shell:
+    bot-name: "mybot"
+```
+
+because `shell` prints to, well, the shell, let's disable logging
+
+```yaml
+logging:
+  console: false
+```
+
+Next, we'll create a basic skill. We'll learn more about this later in the tutorial. Create a file called \_\_init\_\_.py, and import these libraries:
+
+```python
+from opsdroid.skill import Skill
+from opsdroid.matchers import match_regex
+```
+
+Then create a class that inherits from Skill like this:
+
+```python
+class HelloSkill(Skill):
+    pass
+```
+
+After that, add a async `hello` function decorated with `@match_regex(r'hi')`. Essentially, this makes it so that if the user types "hi" this function runs. Let's add a response of "hey" like so:
+```python
+from opsdroid.skill import Skill
+from opsdroid.matchers import match_regex
+
+class HelloSkill(Skill):
+    @match_regex(r'hi')
+    async def hello(self, message):
+        await message.respond('Hey')
+```
+Then add your skill to you yaml file like this:
+
+```yaml
+skills:
+  - name: exampleskill
+    path: /path/to/my/skill.py
+    # Or /path/to/my/skill/ if you created a directory
+    # with an __init__.py file in it
+```
+Amazing! You have created a bot! Run it with `opsdroid start`.
```

Comment:
Keep going here to explain what happens next.

> Then type "hi" and the bot should respond with "Hey"

Then have a sentence saying "For more examples of skills see the examples section".

Reply:
> This is a great start.
> 
> I think you should rename this to `quickstart.md` and add it to the toctree at the very top.
> 
> I would also try and cut out anything unnecessary and link to the rest of the docs as much as possible.

I thought that maybe this could be continued as a more hands-on version of the example section, and maybe we would add additional connector, and parser examples as well as having more skill pages, to make the docs more beginner-friendly.

Reply:
I will add the examples

Reply:
With the examples I basically want a gallery of loads of different things you could do with opsdroid. Mostly to inspire people but also to give copy paste snippets that you can modify and merge into whatever you want.

The hardest thing for a new user is having a blank canvas.

Reply:
> This is a great start.
> 
> I think you should rename this to `quickstart.md` and add it to the toctree at the very top.
> 
> I would also try and cut out anything unnecessary and link to the rest of the docs as much as possible.

@jacobtomlinson added it to toctree, but I am getting an error. I am sure I misconfigured it. Could you try?